packages/core/src/providers/NetworkActivator.tsx

```diff
@@ -4,7 +4,11 @@ import { useEthers } from '../hooks'
 import { InjectedConnector } from '@web3-react/injected-connector'
 import { useConfig } from './config/context'
 
-export function NetworkActivator() {
+interface INetworkActivator {
+  autoConnect: boolean
+}
+
+export function NetworkActivator({ autoConnect }: INetworkActivator) {
   const { activate, account, chainId: connectedChainId, active, connector } = useEthers()
   const { supportedChains, readOnlyChainId, readOnlyUrls } = useConfig()
```

Comment:
Instead of grabbing `autoConnect` through props it could be just taken from `useConfig` in `NetworkActivator`, but in general lgtm.